go.mod

```diff
@@ -15,6 +15,7 @@ require (
 	github.com/aws/smithy-go v1.20.2
 	github.com/gabriel-vasile/mimetype v1.4.2
 	github.com/golang-jwt/jwt/v5 v5.2.1
+	github.com/google/uuid v1.6.0
```

Comment:
I prefer we didn't include a new library. If it is for tests only - I think we can live with our impl only.

Reply:
I was using it as it has the SQL scanner/valuer interfaces; if you're ok with it I can extend your existing UUID definition for testing purposes and add the scanner valuer methods

Reply:
Removed